src/main/java/org/zalando/pazuzu/feature/FeatureRepositoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.pazuzu.feature;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+public class FeatureRepositoryImpl implements FeatureRepositoryCustom {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Override
+    public List<Feature> getFeatures(int offset, int limit) {
```

Comment:
It is not clear how to access entityManager from default interface method.
Here are some some additional arguments for not using default methods: http://stackoverflow.com/questions/26543612/should-i-use-java-8-default-methods-for-manually-implemented-spring-data-reposit#26563841